src/commands/BootstrapCommand.js

```diff
@@ -49,24 +50,24 @@ export default class BootstrapCommand extends Command {
       const linkSrc = dependency.location;
       const linkDest = path.join(pkg.nodeModulesLocation, dependency.name);
 
-      this.createLinkedDependency(linkSrc, linkDest, dependency.name, done);
+      this.createLinkedDependency(linkSrc, linkDest, dependency.name, (err) => {
+        if (!err && dependency._package.bin) {
+          const srcBinLocation = path.join(linkSrc, dependency._package.bin);
```

Comment:
Actually, `dependency.bin` [isn't guaranteed to be a string](https://github.com/lerna/lerna/pull/192/files#r68466171).

Could normalize inside the getter to always return an object.  Else the logic here will need some rework.